ubuntu-16.04-blank/.gitattributes

```diff
@@ -0,0 +1 @@
+ubuntu-16.04-lts-blank.qcow2.bz2 filter=lfs diff=lfs merge=lfs -text
```

Comment:
Wouldn't it be better to have all of the LFS-related lines in one file (`.gitattributes` in the root)?

Reply:
Yes, I have added another commit to fix this.